sympy/integrals/tests/test_integrals.py

```diff
@@ -1288,7 +1288,7 @@ def test_issue_6253():
     assert integrate((sqrt(1 - x) + sqrt(1 + x))**2/x, x, meijerg=True) == \
         Integral((sqrt(-x + 1) + sqrt(x + 1))**2/x, x)
 
-
+@XFAIL
 def test_issue_4153():
     assert integrate(1/(1 + x + y + z), (x, 0, 1), (y, 0, 1), (z, 0, 1)) in [
         -12*log(3) - 3*log(6)/2 + 3*log(8)/2 + 5*log(2) + 7*log(4),
```

Comment:
We need to investigate what causes the test failures rather than marking them as XFAIL. In this case the output still seems to be correct but just in a different form:
```python
In [6]: i = integrate(1/(1 + x + y + z), (x, 0, 1), (y, 0, 1), (z, 0, 1))

In [7]: i
Out[7]: 
  27⋅log(3)                      
- ───────── + 6⋅log(2) + 8⋅log(4)
      2                          

In [8]: i2 = -12*log(3) - 3*log(6)/2 + 3*log(8)/2 + 5*log(2) + 7*log(4)

In [9]: i2
Out[9]: 
             3⋅log(6)   3⋅log(8)                      
-12⋅log(3) - ──────── + ──────── + 5⋅log(2) + 7⋅log(4)
                2          2                          

In [10]: N(i)
Out[10]: 0.417972075299316

In [11]: N(i2)
Out[11]: 0.417972075299316
```
I don't understand why the form has changed at all though because it implies that somewhere a different gcd was computed but the gcd should be the same regardless of which algorithm is used to compute it.